src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -2033,23 +2039,22 @@ theorem msb_append {x : BitVec w} {y : BitVec v} :
     simp [h, q, t, BitVec.msb, getMsbD]
 
 @[simp] theorem append_zero_width (x : BitVec w) (y : BitVec 0) : x ++ y = x := by
-  ext
-  rw [getLsbD_append] -- Why does this not work with `simp [getLsbD_append]`?
-  simp
+  ext i ih
+  rw [getElem_append] -- Why does this not work with `simp [getLsbD_append]`?
```

Comment:
I think we need to update `getElem_append`'s right-hand-side.

Reply:
Fixed.